File: spaceship_crm/scala/spaceship_crm_scala_spring_rest_jpa/src/main/scala/com/spaceship/crm/bean/LoginResponseBean.scala
/**
 * Author veni.vidi.dev (veni.vidi.dev@gmail.com)
 */
package com.spaceship.crm.bean

import com.spaceship.crm.entity.User

import scala.beans.BeanProperty

class LoginResponseBean {
  @BeanProperty var user: User = _
  @BeanProperty var token: String = _

  def this(user: User, token: String) {
    this()
    this.user = user
    this.token = token
  }
}

File: spaceship_crm/java8/spaceship_crm_java8_rest_jooq/src/main/java/com/spaceship/crm/bean/LoginResponseBean.java
/**
 * Author veni.vidi.dev (veni.vidi.dev@gmail.com)
 */
package com.spaceship.crm.bean;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class LoginResponseBean {
    private UserBean user;
    private String token;
}
